client/components/Reports/SummarizeTestPlanReport.jsx

```diff
@@ -307,8 +307,8 @@ SummarizeTestPlanReport.propTypes = {
                                 PropTypes.shape({
                                     id: PropTypes.string.isRequired,
                                     text: PropTypes.string.isRequired,
-                                    otherUnexpectedBehaviorText:
-                                        PropTypes.string
+                                    impact: PropTypes.string.isRequired,
+                                    details: PropTypes.string.isRequired
                                 }).isRequired
                             ).isRequired
                         }).isRequired
```

Comment:
Because of where this comment is made, I don't follow quite follow which page you're proposing that for. Since it's already being shown on `/report/<id>/targets/<id>` for where this comment was linked. Or do you mean for `/report/<id>`?

If it's for `/report/<id>`, it was a requirement of #738 to not display it in the table anymore, as it previously was. I'm all for showing explicit information though. Do you have a suggestion on how to surface that?

Reply:
I see, since this was the only change I was assuming that there were no changes to the way unexpected behaviors were displayed. I just verified that the details and impact are being shown so this is a non-issue.